CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 0.25.0-rc.0 / 2022-12-16
+
+* [CHANGE] Change the default `parse_mode` value from `MarkdownV2` to `HTML` for Telegram. #2981
+* [CHANGE] Make `api_url` field optional for Telegram. #2981
+* [CHANGE] Use CanonicalMIMEHeaderKey instead of TitleCasing for email headers. #3080
+* [CHANGE] Reduce the number of notification logs broadcasted between peers by expiring them after (2 * repeat interval). #2982
+* [CHANGE/BUGFIX] Fail configuration loading if `api_key` and `api_key_file` are defined at the same time. #2910
```

Comment:
We generally don't mix, so this should be a BUGFIX only I think

Reply:
Ditto, this looks like a bugfix.

Reply:
I thought that Alertmanager v0.24.0 would have accepted `api_key` and `api_key_file` but in fact setting `api_key_file` failed the configuration loading...